Loop/Managers/LoopDataManager.swift

```diff
@@ -1482,6 +1510,19 @@ extension LoopDataManager {
     }
 }
 
+extension LoopDataManager {
+    // This should ONLY be used for testing
+    public func updateAndGetRecommentations(completion: @escaping ([PredictedGlucoseValue]?, (recommendation: TempBasalRecommendation, date: Date)?, (recommendation: BolusRecommendation, date: Date)?) -> Void) {
+        self.dataAccessQueue.async {
+            do {
+                try self.update()
+                completion(self.predictedGlucose, self.recommendedTempBasal, self.recommendedBolus)
+            } catch let error {
+                self.logger.error("%{public}@", String(describing: error))
+            }
```

Comment:
This adds a public way to access the `update` function. I'm open to changing this, but the reason I did it this way is because it allows for a completion handler that can return all the needed information to carry out the LDM effects tests

Reply:
I was able to actually just use the `getLoopStatus` to get what I need!